### PR TITLE
Update org.standardnotes.standardnotes.metainfo.xml

### PR DESCRIPTION
**Pull request by baptiste-grob**
Hey there! I'm Baptiste, and I work at Standard Notes.

It's nice to see you managed to build a flatpak of the app!

Unfortunately we're not in a position to maintain it at the moment. However, we know there are many people who would like to use the app as a flatpak, so this PR adds "(Unofficial)" to relevant description fields to avoid any confusion with what users can expect in terms of support.

Thanks!

### CODE REVIEW

File: org.standardnotes.standardnotes.metainfo.xml [hunk 1]
```diff
@@ -33,6 +33,9 @@
   </categories>
 
   <description>
+    <p><em>(Note: This release is built by a 3rd party from official Standard Notes releases. Standard Notes is
```
Comment (proletarius101) [inline org.standardnotes.standardnotes.metainfo.xml:36]: Sorry for the late response!

Please move this to the last line of the description, and rephrase it like `NOTE: This release is not verified by, affiliated with, or supported by Standard Notes Ltd.` which is a standard statement as appeared in https://flathub.org/apps/details/com.getpostman.Postman

https://flathub.org/apps/details/com.getpostman.Postman

File: org.standardnotes.standardnotes.metainfo.xml [hunk 2]
```diff
@@ -2,10 +2,10 @@
 <component type="desktop-application">
   <id>org.standardnotes.standardnotes</id>
 
-  <name>Standard Notes</name>
-  <summary>A simple and private place for your notes, thoughts, and life's work</summary>
+  <name>Standard Notes (Unofficial)</name>
```
Comment (proletarius101) [inline org.standardnotes.standardnotes.metainfo.xml:5]: There is no app listed in flathub having an `unofficial` suffix. I think it's sufficient to have the statement in the description/summary.

**Review by proletarius101 — CHANGES_REQUESTED** [review]

**Comment by proletarius101** [comment]
Thanks!

**Comment by barthalion** [comment]
Hey @baptiste-grob, I'm one of Flathub co-founders/administrators/infrastructure engineers. Just a heads-up that I decided to drop `(Unofficial)` from the summary (it's the short blurb visible under application name) as it's very not standard and just doesn't look well. However, I moved the "not affiliated" note to the top of the regular description so it should be better visible on the website and native frontends like GNOME Software. I hope that's fine with you.

**Pull request merged**